english/lego/lego.md

```diff
@@ -2,7 +2,7 @@ The mallow man seems nice so you chat to him after walking up to him.
 
 He decides he likes you too, so he gets out his Lego's.
 
-He's got Star Wars, Lego Movie, space, technics, oh so much lego and you play with it all!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.